sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java

```diff
@@ -20,13 +20,14 @@
 import static org.sormas.e2etests.pages.application.dashboard.Contacts.ContactsDashboardPage.CONFIRMED_COUNTER_ON_CONTACTS_DASHBOARD;
 import static org.sormas.e2etests.pages.application.dashboard.Contacts.ContactsDashboardPage.CONFIRMED_COUNTER_ON_CONTACTS_DASHBOARD_DE;
 import static org.sormas.e2etests.pages.application.dashboard.Contacts.ContactsDashboardPage.UNDER_FU_CHART_ON_CONTACTS_DASHBOARD;
-import static org.sormas.e2etests.pages.application.dashboard.Surveillance.ContactsDashboardPage.CONTACTS_COVID19_COUNTER;
+import static org.sormas.e2etests.pages.application.dashboard.Surveillance.ContactsDashboardPage.*;
```

Comment:
avoid doing this, we have a lot of locators with same name, from different pages. The wildcard import brought a lot of issues with Allure report, cucumber wasn't able to distinguish locators.